wasm/src/bindings_c2d.cpp

```diff
@@ -77,7 +77,25 @@ class RendererWrapper : public wrapper<rive::Renderer>
 
     void restore() override { call<void>("restore"); }
 
-    void transform(const rive::Mat2D& transform) override { call<void>("transform", transform); }
+    void transform(const rive::Mat2D& transform) override
+    {
+        call<void>("transform",
+                   transform.xx(),
+                   transform.xy(),
+                   transform.yx(),
+                   transform.yy(),
+                   transform.tx(),
+                   transform.ty());
+    }
+
+    void align(RendererWrapper& self,
```

Comment:
its the existing `align` function but want to make sure we use this `RendererWrapper`'s  `transform` method above, rather than the existing CPP implementation in rive-cpp in `renderer.cpp` but I think we can mark this as `override` to make it clearer